presto-main/src/main/java/io/prestosql/sql/analyzer/FeaturesConfig.java

```diff
@@ -221,6 +221,7 @@ public boolean isOmitDateTimeTypePrecision()
     }
 
     @Config("deprecated.omit-datetime-type-precision")
+    @ConfigDescription("Enable compatibility mode for legacy clients when rendering datetime type names with default precision in information_schema tables.")
```

Comment:
It's not just for information_schema. It affects DESCRIBE INPUT/OUTPUT and the jdbc.columns table. I would just get rid of `in information_schema tables`.

Reply:
Thanks, fixed. Maybe we want to fix RN wording as well then?